src/splitInteger.test.js

```diff
@@ -4,18 +4,23 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(10, 5)).toEqual([2, 2, 2, 2, 2]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
+  const value = Math.floor(Math.random() * 100) + 1;
+  const result = splitInteger(value, 1);
 
+  expect(result).toEqual([value]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
+  const result = splitInteger(19, 3);
 
+  expect(result.sort()).toEqual([6, 6, 7]);
```

Comment:
Consider verifying if the `splitInteger` function is supposed to return sorted parts by default. If so, sorting the result before the expectation might not be necessary. Ensure the function's behavior matches the test's expectations.